suites/upgrade/infernalis/older/3-upgrade-sequence/upgrade-mon-osd-mds.yaml

```diff
@@ -0,0 +1,46 @@
+meta:
+   - desc: |
+      'upgrade the ceph cluster nodes' - 
+      mon.a and mon.b are the two cluster nodes, 
+      upgrading them upgrades everything there.
+      restart order
+      mon.a(b,c) wait 60 sec between restarts 
+      mds.a wait 60 sec between restarts
+      osd.0(1,2,3,4,5) wait 30 sec between restarts
```

Comment:
the time between restarts isn't really significant - the order is the important part.

I'd try to make a shorter description, like 'upgrade the ceph cluster, ordering: mon, mds, osd